configs/enet/enet_cityscapes_1024x512_adam_0.002_80k.yml

```diff
@@ -0,0 +1,55 @@
+iters: 80000
```

Comment:
if you only change the training iter, plz elaborate on why this is necessary. Normally, we do as the original implementation

Reply:
Done.